st2common/st2common/models/db/action.py

```diff
@@ -215,6 +209,9 @@ class ActionAliasDB(stormbase.StormBaseDB):
     formats = me.ListField(
         field=me.StringField(),
         help_text='Possible parameter formats that an alias supports.')
+    file_uri = me.StringField(
```

Comment:
Oh thats coz we are adding a pack attribute to RuleDB. Should've been clearer. Need to finish up remaining changes ofcourse.